src/dxtbx/nexus/__init__.py

```diff
@@ -245,6 +245,7 @@ def get_dxtbx_scan(
 def get_dxtbx_detector(
     nxdetector: nxmx.NXdetector,
     wavelength: float,
+    nxdata: nxmx.NXdata = None,
```

Comment:
```suggestion
    nxdata: nxmx.NXdata | None = None,
```
No implicit-optional